src/numberGenerator.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const generateRandomNumber = () => {
+  const min = 1000;
+  const max = 9999;
+
+  const randomNumber = Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
The generated number may have repetitive digits, which is not allowed in Bulls and Cows game. Consider generating a 4-digit number with unique digits.